packages/gateway/plugins/auth.ts

```
@@ -95,19 +98,15 @@ const validateUser: ValidateUserFn<UserType> = ({ user, fieldDirectives, parentT
     return;
   }
 
-  const role = fieldDirectives?.auth?.find(arg => 'role' in arg)?.role;
+  /* eslint-disable @typescript-eslint/no-explicit-any */
+  const role = fieldDirectives?.auth?.find((arg: any) => 'role' in arg)?.role;
   const acceptableRoles = getAcceptableRoles(role);
 
   if (user.role && acceptableRoles.includes(user.role)) {
     return;
   }
+  if (user.role == 'ADMIN') {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)